src/components/folderPageComponents/Kebab.js

```diff
@@ -0,0 +1,79 @@
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import kebab from '../../assets/Images/kebab.png';
+import {
+  PopOverWrapper,
+  PopOver,
+  PopOverButton,
+} from '../../styles/styledComponents/common';
+import { FolderPageContext } from '../../context/FolderPageContext';
+import { cardLinkCount } from '../../assets/utils/cardLinkCount';
+
+export const Kebab = ({ currentCardLink }) => {
+  const [openPopOver, setOpenPopOver] = useState(false);
```

Comment:
```suggestion
  const [isOpenPopOver, setIsOpenPopOver] = useState(false);
```

[A/HC/LC Pattern](https://jopemachine.github.io/2021/10/10/Function-Naming-Pattern-A-Hc-Lc-Pattern/)으로 네이밍 컨벤션을 가져가보는건 어떨까요?